### DISCUSSION TITLE
This week in React - newsletter

### DISCUSSION THREAD

**Discussion by pwierzbieniec** [Git-Prove/GitProve-Projects#6] [Resources & Learning]
Weekly newsletter that collects the most important changes, features in React Ecosystem.

https://thisweekinreact.com/

**Comment by LachPawel** [comment]
After working with React for a couple of years, finishing bunch of tutorials including Epic React by Kent C. Dodds I still struggled with the useEffect hook from time to time - this [https://www.youtube.com/watch?v=QQYeipc_cik](https://www.youtube.com/watch?v=QQYeipc_cik) is by far the best explanation of useEffect I have ever seen also covered with React 18 version. Strongly recommend that! :)
